vj4/model/adaptor/training.py

```diff
@@ -8,16 +8,17 @@
 
 
 @argmethod.wrap
-async def add(domain_id: str, title: str, content: str, owner_uid: int, dag=[]):
+async def add(domain_id: str, title: str, intro: str, content: str, owner_uid: int, dag=[]):
```

Comment:
That's Ok. By the way shall we upgrade existing content into another field, or just do a field mapping here to prevent migrating current data?